framework/test/unit/engine/endpoint/txpool.spec.ts

```diff
@@ -210,14 +225,36 @@ describe('txpool endpoint', () => {
 	});
 
 	describe('getTransactionsFromPool', () => {
```

Comment:
Would it be relevant to add another test case to verify if schema is validated, something of sort 'should throw/fail if provided address in not a valid lisk32 address'?

Reply:
I believe the unit tests for schema validator should focus on testing that every type is correctly validated, so that the code which uses schema validator doesn't have to. Otherwise, we would bloat many unit tests with schema tests.

@ishantiw what is the convention about this?